CHANGES.rst

```diff
@@ -5,7 +5,11 @@ master (unreleased)
 -------------------
 
 * Fix infinite recursion with multiple `MonitorField` and `defer()` or `only()`
-on Django 1.10+. Thanks Romain Garrigues. Merge of #242, fixes #241.
+  on Django 1.10+. Thanks Romain Garrigues. Merge of #242, fixes #241.
+* Fix issue #249 when extending `Manager` and `QuerySet` classes to add custom
```

Comment:
I don't have any strong feeling. I guess we could keep using `GH-` for consistency, but I also don't mind switching to the simpler `#`.